tests/models/resource_test.py

```diff
@@ -373,6 +373,17 @@ def test_provisional_user_can_delete_own_resource(self):
             result = test_resource.delete(user=user)
             self.assertTrue(result)
 
+        with self.subTest(user="can't delete"):
```

Comment:
Next time, I wouldn't reuse a subTest label, but it's not the worst thing since unlike assertions in a for loop, you at least have separate line numbers here.

The flip side is that using subTest outside a loop seems like not as strong a choice as a test class with shared setup and then separate test methods for the separate assertions, but it's fine to blend in for now 👍